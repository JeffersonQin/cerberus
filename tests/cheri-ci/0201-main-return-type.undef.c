float main(void) { return 1.; }
