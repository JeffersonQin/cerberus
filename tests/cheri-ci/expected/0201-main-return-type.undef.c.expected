Undefined {ub: "UB004a_incorrect_main_return_type", stderr: "", loc: "<1:1--1:6>"}
